app/Http/Controllers/ProductController.php

```diff
@@ -96,10 +103,17 @@ public function previewEmail(Product $product)
     public function show(Product $product)
     {
         $permissions = [
-            'delete' => Auth::user()->can('delete', $product),
-            'update' => Auth::user()->can('update', $product)
+            'delete' => false,
```

Comment:
Good job for making the users viewing the index without authentication ,which is good for business

Reply:
thanks :)